rotkehlchen/tests/unit/decoders/test_balancer.py

```diff
@@ -0,0 +1,425 @@
+                    hexstring_to_bytes('0xc6a5032dc4bf638e15b4a66bc718ba7ba474ff73000200000000000000000004'),  # noqa: E501
+                    hexstring_to_bytes('0x000000000000000000000000c02aaa39b223fe8d0a0e5c4f27ead9083c756cc2'),  # noqa: E501
+                    hexstring_to_bytes('0x0000000000000000000000006b175474e89094c44da98b954eedeac495271d0f'),  # noqa: E501
+                ],
+            ), EvmTxReceiptLog(
+                log_index=95,
+                data=hexstring_to_bytes('0x00000000000000000000000000000000000000000000000000038d7ea4c68000'),  # noqa: E501
+                address=string_to_evm_address('0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2'),
+                removed=False,
+                topics=[
+                    hexstring_to_bytes('0xe1fffcc4923d04b559f4d29a8bfc6cda04eb5b0d3c460751c2402c5c5cc9109c'),  # noqa: E501
+                    hexstring_to_bytes('0x000000000000000000000000ba12222222228d8ba445958a75a0704d566bf2c8'),  # noqa: E501
+                ],
+            ), EvmTxReceiptLog(
+                log_index=98,
+                data=hexstring_to_bytes('0x00000000000000000000000000000000000000000000000010c0749f9f7289bb'),  # noqa: E501
+                address=string_to_evm_address('0x6B175474E89094C44Da98b954EedeAC495271d0F'),
+                removed=False,
+                topics=[
+                    hexstring_to_bytes('0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef'),  # noqa: E501
+                    hexstring_to_bytes('0x000000000000000000000000ba12222222228d8ba445958a75a0704d566bf2c8'),  # noqa: E501
+                    hexstring_to_bytes('0x00000000000000000000000020a1cf262cd3a42a50d226fd728104119e6fd0a1'),  # noqa: E501
+                ],
+            ),
+        ],
+    )
+
+    dbethtx = DBEvmTx(database)
+    with database.user_write() as cursor:
+        dbethtx.add_evm_transactions(cursor, [transaction], relevant_address=None)
+        decoder = EthereumTransactionDecoder(
+            database=database,
+            ethereum_inquirer=ethereum_manager.node_inquirer,
+            transactions=eth_transactions,
+        )
+        events = decoder.decode_transaction(cursor, transaction=transaction, tx_receipt=receipt)
+
+    assert events == [
+        HistoryBaseEntry(
+            event_identifier=evmhash,
+            sequence_index=0,
+            timestamp=1646375440000,
+            location=Location.BLOCKCHAIN,
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.FEE,
+            asset=A_ETH,
+            balance=Balance(amount=FVal('0.00393701451')),
+            location_label='0x20A1CF262Cd3A42a50D226fD728104119e6fD0a1',
+            notes='Burned 0.00393701451 ETH for gas',
+            counterparty='gas',
+            identifier=None,
+            extra_data=None,
+        ), HistoryBaseEntry(
+            event_identifier=evmhash,
+            sequence_index=1,
+            timestamp=1646375440000,
```

Comment:
In the test file (all the instances of `HistoryBaseEntry` can you fix the type here? Should be `TimesatmpMS`